presto-native-execution/presto_cpp/main/common/Configs.h

```diff
@@ -584,6 +584,10 @@ class SystemConfig : public ConfigBase {
   static constexpr std::string_view kDriverMaxPagePartitioningBufferSize{
       "driver.max-page-partitioning-buffer-size"};
 
+  // Specifies the default Presto namespace prefix.
+  static constexpr std::string_view kPrestoDefaultNamespacePrefix{
+      "presto.default-namespace"};
```

Comment:
@pdabre12 : Should this be just "presto.default" instead of "presto.default-namespace"

Reply:
This is the config name. There are no configs that start with `presto.
How about a more specific name like `default-object-namespace` or something like that?

Reply:
Thanks @czentgr .
Yes it is the config name.  There is a config that start with `presto.` `(presto.version)`.  I thought it might make sense to put this config under `presto.` 
`default-object-namespace` seems fine , but what about `default-presto-object-namespace` or `default-presto-namespace` ?

Reply:
Thanks. I overlooked the one that actually has it in the name :).
But the reason it is there is because it is a "shared" config that also exists in Presto itself. The namespace doesn't really exist there as otherwise we could just use the same name for it. And so using the prefix `presto` seems to me it could make things confusing.
Is there any though that this could change dynamically (i.e. via session property)?

Reply:
So, if you take a look at issue https://github.com/prestodb/presto/issues/23385, this is going to be a shared config which is going to be introduced in Presto in a follow-up PR. That's why I think it makes sense to have the prefix `presto` before. 
Let me know what your thoughts are.

Reply:
Ok, then I'm good with the name as it affects both sides, like `presto.version`.